projects/plugins/jetpack/class.jetpack-admin.php

```diff
@@ -43,10 +38,12 @@ public static function init() {
 	/**
 	 * Filter callback to add `no-store` to the `Cache-Control` header.
 	 *
+	 * @deprecated 14.9
```

Comment:
Could you use `$$next-version$$` here? It will be automatically be replaced by the correct version number on release.


```suggestion
	 * @deprecated $$next-version$$
```